### ISSUE TITLE
How to show the correct PSNR

### ISSUE THREAD

**Issue by Xiannnnnnn** [lcwLcw123/BKchallenge#2]
I try to show PSNR like this.
 
gt_image = batch.get("target", batch.get("blurred", batch["source"])).cuda()
 gt_image = np.array(to_image(torch.squeeze(gt_image.float().detach().cpu())))
output_image = np.array(to_image(torch.squeeze(output.float().detach().cpu())))
psnr = calculate_psnr(output_image, gt_image, crop_border=0)

but the average PSNR is 39.2489 not equal your PSNR.
I think I do something wrong, could you help me plz.
thank you
